docs/ruleset_functions/expected_system_type_from_Table_G3_1_1.md

```diff
@@ -0,0 +1,117 @@
+# expected_system_type_from_Table_G3_1_1
+**Schema Version:** 0.0.22  
+
+**Description:** returns the expected system type based on the BAT, the number of floors of the building and the area of the BAT, and climate zone
+
+**Inputs:**
+- **building_area_type** - ("RESIDENTIAL", "PUBLIC_ASSEMBLY", "RETAIL", "HOSPITAL", "OTHER_NON_RESIDENTIAL", "HEATED-ONLY_STORAGE")
+- **number_of_floors** - this is the number of floors in the building integer greater than 0
+- **area_of_BAT** - this is the total area of the building area type
+
+**Returns:**  
+- **result**: a dict indicating the expected system type from table G3_1_1 ("SYS-1", "SYS-2", etc) and a string that indicates how the system was chosen, for example: {"EXPECTED_SYSTEM_TYPE": "SYS-4", "SYSTEM_ORIGIN": "PUBLIC_ASSEMBLY CZ_0_to_3a < 120,000 ft2"} - does not take into account G3.1.1 b-g
+ 
+**Function Call:**
+- **is_CZ_0_to_3a**
```

Comment:
@KarenWGard we're calling multiple functions- number of floors, HVAC BAT- correct? If yes, please do list them here.

Reply:
there are a number of inputs to this function that are generated by other functions.  The reason I have them as inputs is because the function that calls this function (get_expected_baseline_hvac_system_type), needs these values for other calculations